plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -237,6 +359,14 @@ class HealthChecksFormSection extends Component {
               onClick={this.props.onAddItem.bind(this, {value: data.healthChecks.length, path: 'healthChecks'})}>
               <Icon color="purple" id="plus" size="tiny" /> Add Health Check
             </a>
+            <Tooltip
```

Comment:
I'm with @mlunoe so let's get a designer's input on this. @leemunroe what do you think?

In the title:
![](https://cl.ly/1t1A2l01190t/Screen%20Shot%202017-01-24%20at%2011.27.40%20AM.png)

In the help text:
![](https://cl.ly/0E2x33281E20/Screen%20Shot%202017-01-24%20at%2011.27.00%20AM.png)